### DISCUSSION TITLE
Please tell us about your battery

### DISCUSSION THREAD

**Discussion by hif2k1** [hif2k1/battery_sim#60] [General]
We need to add more batteries to the battery_sim so users can setup their battery quicker. Please post the specifications of batteries you are aware of or are simulating. We especially need batteries from outside the UK. To add a battery we need the size (kWh), efficiency (0-1), maximum charging rate (kW) and maximum discharging rate (kW). Many thanks for your contribution!

**Comment by rrozema** [comment]
Enphase IQ Battery 3 specs: https://support.enphase.com/s/article/Encharge-3-Storage-System-Data-Sheet
characteristics: 1-phase AC coupled Lithium Iron phosphate (LFP) battery system
size: 3.36 kWh
efficiency: 89%
max. charging rate: 1280 kW
max. discharging rate: 1.28kVA @ 0.85 .. -0.85 PF = 1.088 .. 1.280 kW (continuous)
max. discharging rate:  1.92KVA @ 0.85 .. -0.85 PF = 1.632 .. 1.92 kW (peak, 10s max)


Enphase IQ Battery 10 specs: https://support.enphase.com/s/article/Encharge-10-Storage-System-Data-Sheet
characteristics: 1-phase AC coupled Lithium Iron phosphate (LFP) battery system
size: 10.08 kWh
efficiency: 89%
max. charging rate: 3840 kW
max. discharging rate: 3.84 kVA @ 0.85 .. -0.85 PF = 3.264 .. 3.84 kW (continuous)
max. discharging rate:  5.70 KVA @ 0.85 .. -0.85 PF = 4.845 .. 5.70 kW (peak, 10s max)


Note 1: Max. charging rate I could not find from the specs. Number i put in was written by an Enphase employee as 1280 kW for 3 and 3840 for 10 model. (https://support.enphase.com/s/question/0D53m000072xMqSCAU/how-much-current-does-encharge-draw-when-charging)
Note 2: I don't exactly know what the power factor means in the conversion from kVA into kWh, specs say however PF is adjustable from leading 0.85 to 0.85 lagging, so I think this means it can be set to 1, resulting in kVA equals kWh.

**Comment by catshapednoodles** [comment]
Sessy: https://www.sessy.nl/#specificaties
size: 5.0 kWh
efficiency: 0.81
maximum charging rate: 2.2 kW
maximum discharging rate: 1.7 kW

I got the efficiency from some notes on a webinar about this battery. There it was said that the efficiency was 90%, but it was later clarified on a Dutch forum (tweakers.net) that this is a one-way efficiency, so to get the full round trip efficiency you have to multiply them to get 0.81/81%.

**Comment by hif2k1** [comment]
Thanks for these, that's great. I'll get them added to a future release.

**Comment by pedromfa** [comment]
**Solax 5.8 kwh master**

![Screenshot 2024-03-13 211529](https://github.com/hif2k1/battery_sim/assets/47584375/f29cbc0a-ada7-4536-ad0d-346c7a3b8d3d)

**Reply by Hamish-Blinx** [reply]
Thanks for the suggestion I will add to it.

**Comment by Jobrr** [comment]
Hi,
The BYD batteries are quite popular as well since they are cheap and can be easily extended afterwards.
Here is a link to their models and specs: 
https://bydbatterybox.com/uploads/downloads/BYD%20Battery-Box%20Premium_Datasheet_HV-AU%20V1.2%20EN-5eec6422498ad.pdf

**Reply by Hamish-Blinx** [reply]
Thanks for the suggestion I will add to them.

**Comment by bderubinat** [comment]
Hi,

Great tool. Just configured it today. My plan is to figure out what size of battery I would be looking for when I can't use the Grid as virtual battery anymore.

Because I'm expecting the battery price to drop until then, I'm looking at a very wide range of capacity (10 to 150 kWh).

I'm also likely to integrate something like single or two Victron Multiplus II in front of the batteries.

Therefore, I added generic batteries with generic capacity / (dis)charging rates: 10kWh/5kW, 10/10, 30/10, 60/10, 90/10, 120/10, 150/10

Yes, 150kWh is massive. I just want it to stretch and figure out impact. Who know how low the pricing can get. It's already "as low" as 2000€ for 10kWh in china (compared to 8000€ for the same 10kWh a few years ago).

**Reply by Hamish-Blinx** [reply]
Maybe one day!
